src/components/page/HeaderBreadcrumb/HeaderBreadcrumb.tsx

```diff
@@ -12,41 +12,36 @@ export interface Page {
 export interface BreadCrumbProps {
   currentPage: Page;
   pages: Page[];
+  className?: string;
 }
 
 /** Configurable Breadcrumbs Component */
-class HeaderBreadcrumb extends React.Component<BreadCrumbProps, {}> {
-  constructor(props: BreadCrumbProps) {
-    super(props);
-  }
+const HeaderBreadcrumb = (props: BreadCrumbProps) => {
+  const { currentPage, pages } = props;
 
-  public render() {
-    const { currentPage, pages } = this.props;
+  const linkList = pages.map((page, key) => {
+    // render breadcrumb items with urls as links or without urls as text nodes
+    let breadCrumbItem: string | JSX.Element;
+    if (page.url && page.url.trim()) {
+      breadCrumbItem = (
+        <Link to={page.url} key={key}>
+          {page.label}
+        </Link>
+      );
+    } else {
+      breadCrumbItem = page.label;
+    }
+    return <BreadcrumbItem key={`reveal-breadcrumb-${key}`}>{breadCrumbItem}</BreadcrumbItem>;
+  });
 
-    const linkList = pages.map((page, key) => {
-      // render breadcrumb items with urls as links or without urls as text nodes
-      let breadCrumbItem: string | JSX.Element;
-      if (page.url && page.url.trim()) {
-        breadCrumbItem = (
-          <Link to={page.url} key={key}>
-            {page.label}
-          </Link>
-        );
-      } else {
-        breadCrumbItem = page.label;
-      }
-      return <BreadcrumbItem key={`reveal-breadcrumb-${key}`}>{breadCrumbItem}</BreadcrumbItem>;
-    });
-
-    return (
-      <div>
-        <Breadcrumb className="reveal-breadcrumb">
-          {linkList}
-          <BreadcrumbItem active={true}>{currentPage.label}</BreadcrumbItem>
-        </Breadcrumb>
-      </div>
-    );
-  }
-}
+  return (
+    <div>
+      <Breadcrumb className={`reveal-breadcrumb${props.className ? ' ' + props.className : ''}`}>
```

Comment:
Nitpick: lets get this prop with the rest while they are being destructured.

Reply:
addressed https://github.com/onaio/reveal-frontend/pull/1194/commits/078fe9203dd24dc48f8dd1322b308855fc2214a3